scripts/task_process/task_proc_wrapper.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+if [ ! -f /etc/enable_status_daemon ]; then
+    echo "/etc/enable_status_daemon file not found, not starting the status daemon and exiting"
+    exit 1
+fi
+
+TIME_BETWEEN_QUERIES=24 #hours
+# SECONDS variable is always being automatically incremented.
+# for the beginning of this script, set it to a big value so that condor_q can be done initially
+SECONDS=$(($TIME_BETWEEN_QUERIES * 3601))
+
+echo "Starting status daemon wrapper"
+# Cluster ID is passed from the dagman_bootstrap_startup.sh and points to the main dag
+CLUSTER_ID=$1
+echo "CLUSTER_ID: $CLUSTER_ID"
+# Sleeping until files in the spool dir are created. TODO - make this smarter
+sleep 60s
+# Loop will exit when the dag code becomes 5 or 6.
+while true
+do
+    
+    if [ $SECONDS -gt $(($TIME_BETWEEN_QUERIES * 3600)) ]; then
+        echo "$SECONDS since last query, requerying."
+        CONDOR_RES=$(condor_q $CLUSTER_ID -af JobStatus)
+        LAST_QUERY_DONE=$(date +"%T")
```

Comment:
what about using date +'%s' to get the seconds since epoch. date +"%T" returns something that likely you'll need to parse